src/en/news/blog/2024/v17-2-8-quincy-released/index.md

```diff
@@ -0,0 +1,1452 @@
+---
+title: "v17.2.8 Quincy released"
+date: "2024-11-18"
+author: "Yuri Weinstein"
+tags:
+  - "release"
+  - "quincy"
+---
+
+This is the eighth backport release in the Quincy series. We recommend all users update to this release.
+
+## Notable Changes
+
+- RADOS: `get_pool_is_selfmanaged_snaps_mode` C++ API has been deprecated
```

Comment:
s/`get_pool_is_selfmanaged_snaps_mode` C++ API/The `get_pool_is_selfmanaged_snaps_mode` C++ API/